### ISSUE TITLE
Re-Enable GitHub Action Runtime Environment

### ISSUE THREAD

**Issue by pavlovcik** [ubiquity/ubiquibot#310]
We should enable support from running the bot natively off of GitHub Actions again, but this time using `npx tsx index.ts` in order to eliminate the complexities around compilation, and then updating the `dist/index.js` file.

The bot should use GitHub Actions as the runtime environment for forks automatically for testing purposes. 

For production we should continue using a server less backend (we are currently using Netlify) this is because, from what I understand, the only way to easily add the bot from the GitHub Marketplace and immediately start using is if it is using an external backend. Otherwise, keeping all of the infrastructure on GitHub is preferred.

**Comment by 0xcodercrane** [comment]
Hmm yes we can enable it actually.

**Comment by 0xcodercrane** [comment]
making a progress on it

**Comment by pavlovcik** [comment]
But the last update was last week. Perhaps it would be in everybody's best interest if you only were assigned and focused on one task at a time?

**Comment by 0xcodercrane** [comment]
resuming the work here

**Comment by pavlovcik** [comment]
Hey we should discuss this before you close and claim it. I expected this to be a huge project to make it run on the netlify backend for production, but then to only run on GitHub Actions for the bounty hunter forks. Can you verify that is the status?

**Comment by 0xcodercrane** [comment]
TL;DR; the status is fine. so we have 2 interfaces; [netlify function](https://github.com/ubiquity/ubiquibot/blob/development/.netlify/functions/webhooks/webhooks.ts) and [github adapter action](https://github.com/ubiquity/ubiquibot/blob/development/adapters/index.ts). Both are being used separately

**Comment by 0xcodercrane** [comment]
The pr is still being tested locally but i think I made a mistake in the merge. reopen it

**Comment by pavlovcik** [comment]
What's the status here?
